patches/chrome-browser-ui-views-side_panel-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
```

Comment:
It's not clear from the commit message that this is a cherry-pick other than the fact that we wouldn't normally do this. I think they normally specify that in the commit message for clarity

Reply:
ex https://github.com/brave/brave-core/pull/30537/commits/32baed2d85e77fc92feca356900f27f454f1832b